articles/azure-arc/servers/manage-vm-extensions.md

```diff
@@ -54,6 +54,7 @@ Arc-enabled servers support moving machines with one or more VM extensions insta
 |Extension |Publisher |Type |Additional information |
 |----------|----------|-----|-----------------------|
 |Azure Defender integrated vulnerability scanner |Qualys |WindowsAgent.AzureSecurityCenter |[Azure Defender’s integrated vulnerability assessment solution for Azure and hybrid machines](../../security-center/deploy-vulnerability-assessment-vm.md)|
+|Microsoft Antimalware Extension |Microsoft.Azure.Security |IaaSAntimalware |[Microsoft Antimalware Extension for Windows](../..//virtual-machines/extensions/iaas-antimalware-windows)
```

Comment:
Also you have a double slash in the path to the article - ../..// - please remove that extra slash in the path and add the extension to the article name - iaas-antimalware-windows.md. Otherwise it won't resolve and 404s.